R/importWaterML2.r

```diff
@@ -53,7 +53,7 @@ importWaterML2 <- function(obs_url, asDateTime=FALSE, tz=""){
     rawData <- obs_url
     doc <- xmlTreeParse(rawData, getDTD = FALSE, useInternalNodes = TRUE)
   } else {
-    doc <- xmlTreeParse(getWebServiceData(obs_url), getDTD = FALSE, useInternalNodes = TRUE)
+    doc <- xmlTreeParse(obs_url, getDTD = FALSE, useInternalNodes = TRUE)
```

Comment:
You could try adding a regx to 
https://github.com/USGS-R/dataRetrieval/blob/master/R/getWebServiceData.R#L40

any content-type with "xml" gets sent to `xmlcontent `